cairo/ethereum/cancun/fork_types.cairo

```diff
@@ -117,3 +118,26 @@ func EMPTY_ACCOUNT() -> Account {
     tempvar account = Account(value=new AccountStruct(nonce=Uint(0), balance=balance, code=code));
     return account;
 }
+
+func Account__eq__(a: Account, b: Account) -> bool {
```

Comment:
this function name looks like we wish we could overwrite `==` sign in cairo for structs 🥹🥹

Reply:
:(